eras/conway/impl/cddl-files/conway.cddl

```diff
@@ -408,6 +408,7 @@ protocol_param_update =
   , ? 30: coin                   ; governance action deposit
   , ? 31: coin                   ; DRep deposit
   , ? 32: epoch                  ; DRep inactivity period
+  , ? 33: coin                   ; MinFee RefScriptCoinsPerByte
```

Comment:
Oops :grimacing: 
I wonder why round trip tests didn't fail, is it normal?

Reply:
Yes, cause we currently don't test that data generated in Haskell conforms with the CDDL spec. In the matter of fact we can't test that right now, because CDDL tests are only as good as ruby generators. However, cuddle will solve that problem for us. :wink:

Reply:
But something I don't understand is:  the ruby tool generates data based on the cddl, so it wouldn't have generated a value for this field.  But the field is not optional in PParams no? (for @Identity).  In the  deserialization it doesn't seem optional either. But it must be, since all these fields in the cddl are optional.. So am I misreading and it's actually optional in the deserialization? 
But don't worry, I will ask you when we speak next, this is not related to the change or PR!

Reply:
It is optional, because this CDDL is for PParamsUpdate, not PParams :wink: